sdk/servicebus/azure-messaging-servicebus/docs/SyncReceiveAndPrefetch.md

```diff
@@ -79,6 +87,15 @@ The lock timeout configured on the service bus entity and `maxMessages` needs to
 
 Refer to [this][prefetchtradeoff] document for the tradeoff when enabling prefetch (or maxMessages > 1).
 
+## The exception/faulted state in ServiceBusReceiverClient
+
+_The exception topic is not directly related to prefetching, but adding this section given exception/faulted state is also scoped in client object-level (like prefetch-queue) and impacts the behavior of Iterables from `receiveMessages` calls._
+
+The Client types in Service Bus SDK have built-in retry to recover from retriable errors. A `ServiceBusReceiverClient` object can reach a faulted terminal state if it exhausts the maximum retry or encounters a non-retriable error.
+
+Once a `ServiceBusReceiverClient` object runs into such faulted state, an iteration on Iterable returned by `receiveMessages` will throw the relevant exception; this is also expected behavior from any future Iterable objects that the application may obtain from a faulted client object.
```

Comment:
this section is a little verbosey; can this be shortened (or alternately can you include a concrete exception example)?  It basically just says we throw an exception on all future calls once we're in an error state.

Reply:
Yes, I have a task to add a sample for this scenario in our sample directory. I'll shorten the text in this pr and create+link sample as a follow-up.